pkg/pluginmanager/manager.go

```diff
@@ -733,7 +737,17 @@ func installOrUpgradePlugin(p *discovery.Discovered, version string, installTest
 	}
 
 	// Log message based on different installation conditions
-	logPluginInstallationMessage(p, version, plugin != nil, isPluginAlreadyInstalled)
+	installingMsg, installedMsg := getPluginInstallationMessage(p, version, plugin != nil, isPluginAlreadyInstalled)
+
+	var spinnerErr error
+	var spinner component.OutputWriterSpinner
+	if enableSpinner {
+		// Initialize the spinner
+		spinner, spinnerErr = component.NewOutputWriterSpinnerWithOptions(os.Stdout, "text", installingMsg, true, nil)
```

Comment:
Shouldn't hard-code "text" but use the constant from TPR (if we keep it).

Reply:
Agree, fixed.